requirements/edx/github.txt

```diff
@@ -90,7 +90,7 @@ git+https://github.com/edx/xblock-utils.git@v1.0.2#egg=xblock-utils==1.0.2
 -e git+https://github.com/edx-solutions/xblock-google-drive.git@138e6fa0bf3a2013e904a085b9fed77dab7f3f21#egg=xblock-google-drive
 -e git+https://github.com/edx/edx-reverification-block.git@0.0.5#egg=edx-reverification-block==0.0.5
 git+https://github.com/edx/edx-user-state-client.git@1.0.1#egg=edx-user-state-client==1.0.1
-git+https://github.com/edx/edx-proctoring.git@0.12.11#egg=edx-proctoring==0.12.11
+git+https://github.com/edx/edx-proctoring.git@0.12.12#egg=edx-proctoring==0.12.11
```

Comment:
@douglashall Just looking at this while I'm rebasing to get things installed on a sandbox - should the final `0.12.11` be updated as well?